app/src/main/java/com/eveningoutpost/dexdrip/LibreAlarmReceiver.java

```diff
@@ -137,7 +137,7 @@ public void run() {
                                     processReadingDataTransferObject(object);
                                 } catch (Exception e) {
                                     Log.wtf(TAG, "Could not process data structure from LibreAlarm: " + e.toString());
-                                    JoH.static_toast_long("LibreAlarm data format appears incompatible!? protocol changed?");
```

Comment:
The english wording for this string has changed recently which I think is resulting in the merge conflict.